io-ballerina/src/io/bytes/channel_byte_io.bal

```diff
@@ -16,11 +16,11 @@
 
 import ballerina/lang.'value;
 
-# Read the entire channel content as a byte array.
```

Comment:
The description should always have a full stop at the end. It is only for param/return descriptions that we omit the full stop if it is a single sentence.

This is what is mentioned in https://ballerina.io/learn/documenting-ballerina-code/#writing-ballerina-documentation too. But I guess using "However" in the second sentence there has made this a bit unclear. 

@praneesha, shall we rephrase it?